src/moveit_studio_agent_utils/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 3.16.3)
+project(moveit_studio_agent_utils)
+
+find_package(ament_cmake REQUIRED)
+find_package(moveit_studio_agent_msgs REQUIRED)
+find_package(moveit_studio_behavior_interface REQUIRED)
+find_package(moveit_studio_behavior_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(std_msgs REQUIRED)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+    moveit_studio_agent_msgs
+    moveit_studio_behavior_interface
```

Comment:
Are all the deps here needed or just the agent msgs?

Reply:
They definitely were not all needed. Before I had run into some errrors but this time it all just worked, so now it's the reduced set.